### DISCUSSION TITLE
Eurang팀 페어프로그래밍 일정 투표

### DISCUSSION THREAD

**Discussion by kuro11pow2** [euphony-io/euphony#69] [General]
![image](https://user-images.githubusercontent.com/47289893/135261838-ad525ebd-a586-4159-9530-bd60562b0412.png)

줌 + vscode live share 확장 사용하여 페어프로그래밍

## 빌드, 최종 테스트
이혜성 (테스트케이스)
박준규

## UTF16
김서현
박준규

## UTF8
이도현
민상연

**Comment by judemin** [comment]
저는 일단 시간대는 저녁이 좋을 것 같습니다!

**Reply by judemin** [reply]
20시나 22시쯤이 적당하다고 생각합니다

**Reply by aiclaudev** [reply]
저도 그 시간이 좋습니다!
저는 일단 금요일 괜찮은데, 요일은 언제가 괜찮으세요?

**Reply by judemin** [reply]
이번주 금요일은 조금 힘들 것 같습니다 ㅠㅜ 아니면 아예 토요일 오후는 어떠세요?? 13시쯤이나

**Reply by aiclaudev** [reply]
토요일 오후도 좋습니다! 2시간정도 진행한다고하면, 12시부터 가능하실까요?! ㅠㅠ

**Reply by judemin** [reply]
넵 좋습니다 그럼 토요일 12시부터 14시까지 진행할까요??

**Reply by judemin** [reply]
제가 줌 링크 파고 카카오톡으로 따로 연락드리겠습니다!

**Reply by aiclaudev** [reply]
넵 알겠습니다!

**Comment by kuro11pow2** [comment]
## UTF16
@seoh02h  
서현님 내일 오후 8 ~ 10시 괜찮으신가요?

**Reply by seohy02** [reply]
넵!!! 가능합니다!

**Reply by kuro11pow2** [reply]
그럼 브랜치 만들고 드라이버 준비해두겠습니다!! 완성되면 알려드릴게요 브랜치를 지우고 만들고 그래서 충돌날 것 같은데 force pull 해주세요!

**Reply by seohy02** [reply]
넵!! 감사해요! vscode live share 한 번 써보고 있을게요!

**Reply by kuro11pow2** [reply]
저희가 사용할  폴더를 develop 브랜치에 만들었습니다.
프로젝트 루트/_cpp_test/utf16/ 경로를 vscode로 열어주시면 됩니다!!

**Reply by seohy02** [reply]
넵넵!! 👍👍👍👍

**Reply by kuro11pow2** [reply]
@seoh02h 이따 8시에 구글 밋 초대 드릴게요!

**Reply by seohy02** [reply]
넵!

**Comment by kuro11pow2** [comment]
## 빌드, 최종 테스트
@ycs1m1yk 
혜성님 저희는 UTF16 완성되는대로 멘토님 의견 듣고 바로 테스트 시작해봐요!

**Reply by ycs1m1yk** [reply]
넵넵 알겠습니다!

**Comment by designe** [comment]
오 vscode 로 페어프로그래밍을 !!
위클리 미팅때 페어프로그래밍 사례 전파 세미나 한번 하시져 :)

**Reply by kuro11pow2** [reply]
헐ㅋㅋㅋㅋㅋㅋ 열심히 준비해봐야겠네여 🚀
